src/main/java/org/apache/flink/state/benchmark/StateBenchmarkBase.java

```diff
@@ -52,10 +56,21 @@
     final ThreadLocalRandom random = ThreadLocalRandom.current();
 
     @Param({"HEAP", "ROCKSDB"})
-    protected StateBackendBenchmarkUtils.StateBackendType backendType;
+    private StateBackendBenchmarkUtils.StateBackendType backendType;
+
+    @Param({"/tmp"})
```

Comment:
Will this be behaving in the same/similar way as it used to be? What if on the given machine there is no `/tmp` directory or the JVM process doesn't have an access to it? I think the old `null` value was more resilient?

Reply:
It's indeed have different behavior, because it's depends on the `java.io.tmpdir` property. 
The `@param` do not accept `null`, so I use `not_set` as default value . 

```
# Warmup: 10 iterations, 1 s each
# Measurement: 10 iterations, 1 s each
# Timeout: 10 min per iteration
# Threads: 1 thread, will synchronize iterations
# Benchmark mode: Throughput, ops/time
# Benchmark: org.apache.flink.state.benchmark.ValueStateBenchmark.valueAdd
# Parameters: (backendType = HEAP, stateDataDirPath = not_set)
```